vexim/adminuserchange.php

```diff
@@ -30,7 +30,13 @@
     <link rel="stylesheet" href="style.css" type="text/css">
     <script src="scripts.js" type="text/javascript"></script>
   </head>
-  <body onLoad="document.userchange.realname.focus()">
+  <script language='JavaScript' type='text/JavaScript'>
```

Comment:
no need for `language='JavaScript'`, and `text/javascript` should go in lowercase.